### ISSUE TITLE
FileNotFoundException .../META-INF/LICENSE (Is a directory)

### ISSUE THREAD

**Issue by skolberg** [pfn/android-sdk-plugin#230]
After updating plugin version from 1.5.7 to 1.5.9, I get the following exception in ```apkBuild```:

```
java.io.FileNotFoundException: <multi-project>/<project>/target/android/intermediates/resources/META-INF/LICENSE (Is a directory)
	at java.io.FileOutputStream.open(Native Method)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:221)
	at sbt.Using$$anonfun$fileOutputStream$1.apply(Using.scala:72)
	at sbt.Using$$anonfun$fileOutputStream$1.apply(Using.scala:72)
	at sbt.Using$$anon$2.openImpl(Using.scala:65)
	at sbt.OpenFile$class.open(Using.scala:43)
	at sbt.Using$$anon$2.open(Using.scala:64)
	at sbt.Using$$anon$2.open(Using.scala:64)
	at sbt.Using.apply(Using.scala:23)
	at android.Packaging$$anonfun$collectNonAndroidResources$4$$anonfun$apply$17$$anonfun$apply$18$$anonfun$apply$19.apply(Packaging.scala:212)
	at android.Packaging$$anonfun$collectNonAndroidResources$4$$anonfun$apply$17$$anonfun$apply$18$$anonfun$apply$19.apply(Packaging.scala:209)
	at sbt.Using.apply(Using.scala:24)
	at android.Packaging$$anonfun$collectNonAndroidResources$4$$anonfun$apply$17$$anonfun$apply$18.apply(Packaging.scala:209)
	at android.Packaging$$anonfun$collectNonAndroidResources$4$$anonfun$apply$17$$anonfun$apply$18.apply(Packaging.scala:208)
	at scala.collection.mutable.LinkedHashSet.foreach(LinkedHashSet.scala:87)
	at android.Packaging$$anonfun$collectNonAndroidResources$4$$anonfun$apply$17.apply(Packaging.scala:208)
	at android.Packaging$$anonfun$collectNonAndroidResources$4$$anonfun$apply$17.apply(Packaging.scala:207)
	at sbt.Using.apply(Using.scala:24)
	at android.Packaging$$anonfun$collectNonAndroidResources$4.apply(Packaging.scala:207)
	at android.Packaging$$anonfun$collectNonAndroidResources$4.apply(Packaging.scala:205)
	at scala.collection.mutable.HashMap$$anon$1$$anonfun$foreach$2.apply(HashMap.scala:102)
	at scala.collection.mutable.HashMap$$anon$1$$anonfun$foreach$2.apply(HashMap.scala:102)
	at scala.collection.mutable.HashTable$class.foreachEntry(HashTable.scala:226)
	at scala.collection.mutable.HashMap.foreachEntry(HashMap.scala:39)
	at scala.collection.mutable.HashMap$$anon$1.foreach(HashMap.scala:102)
	at android.Packaging$.collectNonAndroidResources(Packaging.scala:205)
	at android.Packaging$$anonfun$apkbuild$3.apply(Packaging.scala:63)
	at android.Packaging$$anonfun$apkbuild$3.apply(Packaging.scala:62)
	at sbt.FileFunction$$anonfun$cached$1.apply(Tracked.scala:186)
	at sbt.FileFunction$$anonfun$cached$1.apply(Tracked.scala:186)
	at sbt.FileFunction$$anonfun$cached$2$$anonfun$apply$3$$anonfun$apply$4.apply(Tracked.scala:200)
	at sbt.FileFunction$$anonfun$cached$2$$anonfun$apply$3$$anonfun$apply$4.apply(Tracked.scala:196)
	at sbt.Difference.apply(Tracked.scala:175)
	at sbt.Difference.apply(Tracked.scala:157)
	at sbt.FileFunction$$anonfun$cached$2$$anonfun$apply$3.apply(Tracked.scala:196)
	at sbt.FileFunction$$anonfun$cached$2$$anonfun$apply$3.apply(Tracked.scala:195)
	at sbt.Difference.apply(Tracked.scala:175)
	at sbt.Difference.apply(Tracked.scala:151)
	at sbt.FileFunction$$anonfun$cached$2.apply(Tracked.scala:195)
	at sbt.FileFunction$$anonfun$cached$2.apply(Tracked.scala:193)
	at android.Packaging$.apkbuild(Packaging.scala:66)
	at android.Tasks$$anonfun$50.apply(tasks.scala:649)
	at android.Tasks$$anonfun$50.apply(tasks.scala:639)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:235)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
```

And here are the contents of the folder 'META-INF/license':
```
drwxr-xr-x  12 Kolberg  staff    408 25 Nov 09:20 .
drwxr-xr-x   8 Kolberg  staff    272 25 Nov 09:20 ..
-rw-r--r--   1 Kolberg  staff   1592 25 Nov 09:20 LICENSE.base64.txt
-rw-r--r--   1 Kolberg  staff  10174 25 Nov 09:20 LICENSE.commons-logging.txt
-rw-r--r--   1 Kolberg  staff  10174 25 Nov 09:20 LICENSE.felix.txt
-rw-r--r--   1 Kolberg  staff  26441 25 Nov 09:20 LICENSE.jboss-logging.txt
-rw-r--r--   1 Kolberg  staff   1592 25 Nov 09:20 LICENSE.jsr166y.txt
-rw-r--r--   1 Kolberg  staff   1465 25 Nov 09:20 LICENSE.jzlib.txt
-rw-r--r--   1 Kolberg  staff  10174 25 Nov 09:20 LICENSE.log4j.txt
-rw-r--r--   1 Kolberg  staff   1732 25 Nov 09:20 LICENSE.protobuf.txt
-rw-r--r--   1 Kolberg  staff   1203 25 Nov 09:20 LICENSE.slf4j.txt
-rw-r--r--   1 Kolberg  staff   1598 25 Nov 09:20 LICENSE.webbit.txt

```

**Comment by pfn** [comment]
can be worked around by doing:

`android.dsl.apkExclude("META-INF/LICENSE")`

**Comment by pfn** [comment]
should be fixed by 0ec9ac5
